x-pack/plugins/apm/public/components/alerting/ui_components/chart_preview/chart_preview_helper.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiLoadingChart } from '@elastic/eui';
+import { EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import moment from 'moment';
+import { i18n } from '@kbn/i18n';
+import { TooltipValue } from '@elastic/charts';
+
+export const tooltipProps = {
+  headerFormatter: (tooltipValue: TooltipValue) =>
+    moment(tooltipValue.value).format('Y-MM-DD HH:mm:ss'),
+};
+
+export const NUM_BUCKETS = 5;
+
+export const TIME_LABELS = {
+  s: i18n.translate('xpack.apm.alerts.timeLabels.seconds', {
+    defaultMessage: 'seconds',
+  }),
+  m: i18n.translate('xpack.apm.alerts.timeLabels.minutes', {
+    defaultMessage: 'minutes',
+  }),
+  h: i18n.translate('xpack.apm.alerts.timeLabels.hours', {
+    defaultMessage: 'hours',
+  }),
+  d: i18n.translate('xpack.apm.alerts.timeLabels.days', {
+    defaultMessage: 'days',
+  }),
+};
+
+export const EmptyContainer: React.FC = ({ children }) => (
```

Comment:
Does this need to be exported? I don't see it used outside this file.

Reply:
It doesn't need to be exported. Updated.